### DISCUSSION TITLE
Change font size

### DISCUSSION THREAD

**Discussion by jackiescanlon** [foxglove/community#68] [Q&A] [answered]
Is there a way to change the default font size on foxglove studio? I'd prefer all the text on the screen to be slightly bigger.

**Comment by jtbandes** [comment, marked as answer]
Currently the best way to do this is to use the built-in zoom functionality provided by the browser/app (usually <kbd>cmd/ctrl</kbd> <kbd>+</kbd>).

We do have an open ticket for better default font sizes: https://github.com/foxglove/studio/issues/1998  But currently the styling architecture of the app makes this hard to do. We hope to improve it in the future but I don't think we have a specific timeline to share yet.

**Reply by jackiescanlon** [reply]
Ah, thank you! For whatever reason `Ctrl` + `+` was not working for me, but I was able to hold down `Alt` to get the File/Edit/View... menu. From the View menu I was able to zoom in that way.
